### DISCUSSION TITLE
Missing submenu for workarounds presets

### DISCUSSION THREAD

**Discussion by Xavientois** [renovatebot/renovate#11971] [Suggest an Idea]
Currently, on the docs, there is a list of included presets with sub-pages for each category. The category for Workarounds is missing from this list despite the fact that the page exists: https://docs.renovatebot.com/presets-workarounds/

![Screen Shot 2021-09-30 at 11 51 02 AM](https://user-images.githubusercontent.com/34867186/135489016-147ef189-bd2a-4bcd-b129-4f3274d8e176.png)
f 

Is this something that could be fixed? I was struggling to find the page when I wanted to understand all of the components of the base config.

**Comment by viceice** [comment]
Needs to be added here: 

https://github.com/renovatebot/renovatebot.github.io/blob/d410be626680c7eeea4175c65282f00b609cab0c/mkdocs.yml#L64

**Reply by HonkingGoose** [reply]
I was wondering why we didn't show that! 😄 

I'll go make a PR to get this fixed then... 😄

**Comment by HonkingGoose** [comment]
Problem will be fixed by PR https://github.com/renovatebot/renovatebot.github.io/pull/87

Turns out we had more than 1 missing preset page. 🙈 

Thank you for your report @Xavientois! ❤️ 😄
